src/main/java/com/uber/cadence/workflow/WorkflowInterceptor.java

```diff
@@ -51,6 +51,11 @@ public Promise<WorkflowExecution> getWorkflowExecution() {
     }
   }
 
+  // interceptor executeWorkflow method: no-op interceptors delegate calls to next interceptors used
```

Comment:
This comment should go to WorkflowInterceptorBase. 
Here you might want to stress that this is to match `interceptor:ExecuteWorkflow` behavior in go client.

Reply:
Also, you want to use [javadoc tags](https://www.javatpoint.com/java-comments) to illustrate the meaning of input and return value. Otherwise, it's confusing to have byte[] array input as output.